enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -33,6 +33,30 @@
           </div>
         </div>
         <div class="col-7 border-left">
+
+          % if messages:
+              <div id="messages">
+                % for message in messages:
+                  <div class="alert\
+                    % for tag in message.tags.split(' '):
+                      alert-${ tag }\
+                    % endfor
+                    aria-live="polite">
+                    % if 'success' in message.tags:
+                        <i class="fa fa-check" aria-hidden="true"></i>
+                    % elif 'info' in message.tags:
+                        <i class="fa fa-info-circle" aria-hidden="true"></i>
+                    % elif 'warning' in message.tags:
+                        <i class="fa fa-warning" aria-hidden="true"></i>
+                    % elif 'error' in message.tags:
+                        <i class="fa fa-times" aria-hidden="true"></i>
+                    % endif %}
+                    ${ message }
+                  </div>
+                % endfor
+              </div>
+          % endif
```

Comment:
This would probably make for a good custom template tag, which brings up a good point. We should probably be using Django templates instead of mako templates.